app/src/sharedTest/java/org/oppia/android/app/databinding/StateAssemblerMarginBindingAdaptersTestActivityTest.kt

```diff
@@ -85,10 +84,10 @@ private const val TOLERANCE = 1e-5f
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(
-  application = StateAssemblerMarginBindingAdaptersTest.TestApplication::class,
+  application = StateAssemblerMarginBindingAdaptersTestActivityTest.TestApplication::class,
   qualifiers = "port-xxhdpi"
 )
-class StateAssemblerMarginBindingAdaptersTest {
+class StateAssemblerMarginBindingAdaptersTestActivityTest {
```

Comment:
These renames don't quite make sense to me. This is testing the binding adapters, not the test activity. I suggest reverting the naming changes for this & the other test suites.

Reply:
The change was because we didn't wanted to exempt file as it was discussed with @Sparsh1212 and @anandwana001 on hangouts chat.

Reply:
As per what I can see, there is this Activity added: `StateAssemblerMarginBindingAdaptersTestActivity`.
Now, the test file check expects that we must write a test for this test activity, and the name of that file which tests this activity must be `StateAssemblerMarginBindingAdaptersTestActivityTest`.
Here, above if we are not testing the activity but the binding adapters, then the older naming was correct, as here we aren't testing the activity but the binding adapters. 
Concluding, I suggest to rename this to the earlier version just as Ben said and add a test for the test activity in a new file and please name it as: `StateAssemblerMarginBindingAdaptersTestActivityTest`

Reply:
What I would be testing in this file?

Reply:
> What I would be testing in this file?
> 
> Isn't it is good to add in exempted files?
WDYT @BenHenning ?